platform/core/src/services/HangingProtocolService/HangingProtocolService.test.js

```diff
@@ -14,6 +16,7 @@ const testProtocol = {
   ],
   displaySetSelectors: {
     displaySetSelector: {
+      imageMatchingRules: [],
```

Comment:
Should we not add a test for this new functionality?

Reply:
I agree and I think we should. I've added a TODO for now in the code to do this. I'll try to get to it soon